src/Cli.php

```diff
@@ -224,4 +244,81 @@ public function alterModulesAndThemes(array &$starterKit, array $userInputValues
     return $starterKit;
   }
 
+  /**
+   * Function to get options for acms starterkit questions.
+   *
+   * @param string $command_type
+   *   Must be build/install/both.
+   *
+   * @return array
+   *   List of options.
+   */
+  public function getOptions(string $command_type = ''): array {
+    $questions = $this->getStarterKitsAndQuestions("questions");
+    $options = $output = [];
+    if ($command_type === 'build' || $command_type === 'install') {
```

Comment:
can just check if command_type is not empty instead like:
```suggestion
    if ($command_type) {
```